docs: added few steps to the getting started documentation for face landmark app #101

michellrodriguez08 opened the pull request:
I added the first few steps for the face landmark app for iOS and Android. I reached a small block but as soon as I get that resolved I will add the rest of the steps, just wanted to practice the adding before it's 'perfect'. I also fixed the android configuration bug that Charles mention in the video with the tags and misspells.

cdiddy77 commented on docsite/docs/api_pages/face-landmark-detection.md:
````diff
@@ -9,4 +9,121 @@ Detects the main points of the face and its expressions in real time.
+yarn add react-native-mediapipe react-native-vision-camera react-native-worklets-core
+```
+
+    If correctly you should see a **react-native-mediapipe** dropdown inside **node_modules** 
````
this is not necessary. doing `yarn add` is something that react native developers do every day.

cdiddy77 commented on docsite/docs/api_pages/face-landmark-detection.md:
````diff
@@ -9,4 +9,121 @@ Detects the main points of the face and its expressions in real time.
+
+3. **Terminal Commands:** In your terminal run the following commands
+
+```
+yarn add @shopify/react-native-skia react-native-reanimated
+```
````
Move this up to where we do the other `yarn add` stuff

cdiddy77 reviewed (changes requested):
Just a few changes. This will be great!

michellrodriguez08 commented:
I believe I addressed all the feedback received. I also added the rest of the steps to create the face landmark app. I removed the tabs but in my opinion I feel like it looks a bit crowded but let me know if it makes sense to you or if I should look for a different approach. Thank you!

cdiddy77 reviewed (approved):
ty!
